### ISSUE TITLE
FcmSharp.Exceptions.FcmMessageException: INVALID_ARGUMENT

### ISSUE THREAD

**Issue by Anduil** [bytefish/FcmSharp#22]
`FcmSharp.Exceptions.FcmMessageException: "{
  "error": {
    "code": 400,
    "message": "Request contains an invalid argument.",
    "errors": [
      {
        "message": "Request contains an invalid argument.",
        "domain": "global",
        "reason": "badRequest"
      }
    ],
    "status": "INVALID_ARGUMENT"
  }
}
"`

I have that exception when i call this code:

`        public static FcmMessageResponse Send()
        {
            // Construct the Client:
            using (var client = new FcmClient(Settings))
            {

                // The Message should be sent to the given token:
                var message = new FcmMessage()
                {
                    ValidateOnly = false,
                    Message = new Message
                    {
                        Topic = "develop",
                        Notification = new Notification
                        {
                            Title = "Title",
                            Body = "Body"
                        },
                    }
                };

                // Finally send the Message and wait for the Result:
                var cts = new CancellationTokenSource();
                // Send the Message and wait synchronously:
                return client.SendAsync(message, cts.Token).GetAwaiter().GetResult();
            }
        }`

**Comment by bytefish** [comment]
Maybe "Title" and "Body" are reserved keywords. Did you try different values? Did you check if the settings are read correctly?

**Comment by Anduil** [comment]
1. Yes, I tried to change these words, and nothing has changed.
2. The settings are correct, I think, because it's very difficult to make a mistake there, and there was no exception. Just in case, how can I confirm my settings? Maybe it makes sense to implement a validator for such things?

**Comment by bytefish** [comment]
Indeed, everything looks correct to me. I just got home from work, and I won't be able to look into it today. Maybe there has been a change in the Google API or I changed something to worse with the latest update. 😌 Maybe you could try out some earlier 2.x versions and see if they also have the problem?

**Comment by bytefish** [comment]
Could you cross check if your projectId is correct? You pass it into the settings, see the constructor: https://github.com/bytefish/FcmSharp/blob/master/FcmSharp/FcmSharp/Settings/FileBasedFcmClientSettings.cs. Could you verify, that you are using the correct project id?

**Comment by Anduil** [comment]
@bytefish I fixed it. By inattention, I inserted the wrong project ID that was required. 
I apologize for your time spent.

**Comment by bytefish** [comment]
@Anduil No worries. Great you figured it out and thanks for the Feedback!
